### ISSUE TITLE
Update documentation with additional keys

### ISSUE THREAD

**Issue by pidgeon777** [dail8859/LuaScript#80]
Hello, this is what the doc states:

> Notepad.AddShortcut(name, shortcut, callback)
Registers a Lua function for a shortcut key. The callback is not passed any parameters.
Note: This can only be called during startup.
>
> Parameters:
name (string): the user-friendly name of the shortcut (this is displayed in the menu)
shortcut (string): the modifier and key (e.g. "Ctrl+Alt+Shift+D"). **The key must be A-Z, 0-9, or F1-F12**
callback (function): function to call when the shortcut is triggered

Please add support to the remaining numeric (for example, '*', '-'...) and special keys ('Home', 'Page up', ...).

**Comment by dail8859** [comment]
There is support for some more shortcuts, but looks like it was never documented:

https://github.com/dail8859/LuaScript/blob/84092e732a6d0044086b052caf34dfaf55835870/src/SciTE/LuaExtension.cpp#L512-L532

I recall when I added in these extra shortcuts, that Notepad++ didn't like some of them so I may have tried others such as `*` but it just didn't allow it for some reason.

**Comment by pidgeon777** [comment]
First of all, thanks for the prompt and detailed reply.

I solved the issue by just assigning a blank "" shortcut when calling the `Notepad.AddShortcut(name, shortcut, callback)` instruction, and then by manually assigning the shortcut through the Notepad++ Shortcut Mapper.

I ported my scripts from Python Script into LUA in order to run them with your plugin, and I have to say that now they are extremely fast, also without any annoying 'Python Script is already running' popup in case I activate key combinations too quickly. This works especially good when handling many simultaneous multiple selections.

Congratulations on your project.
